server/parameters/util/Option.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2021 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.core.server.parameters.util;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+import java.util.Optional;
+
+public class Option {
```

Comment:
This class represents a "CLI option". It has been extracted from `ArgsParser` (which used to be called `CommandLine`, prior to this PR).